### PR TITLE
support toast item

### PR DESCRIPTION
**Pull request by Yvent**
support toast item

### CODE REVIEW

**Comment by Yvent** [comment]
@elai950 hi, I submitted some code, I hope it will be useful, can you review it?

File: Sources/AlertToast/AlertToast.swift [hunk 1]
```diff
@@ -682,6 +682,35 @@ public extension View{
         modifier(AlertToastModifier(isPresenting: isPresenting, duration: duration, tapToDismiss: tapToDismiss, offsetY: offsetY, alert: alert, onTap: onTap, completion: completion))
     }
     
+    /// Present `AlertToast`.
+    /// - Parameters:
+    ///   - item: Binding<Item?>
+    ///   - alert: (Item?) -> AlertToast
+    /// - Returns: `AlertToast`
+    func toast<Item>(item: Binding<Item?>, duration: Double = 2, tapToDismiss: Bool = true, offsetY: CGFloat = 0, alert: @escaping (Item?) -> AlertToast, onTap: (() -> ())? = nil, completion: (() -> ())? = nil) -> some View{
```
Comment (nathanfallet) [inline Sources/AlertToast/AlertToast.swift:690]: You might want to make `Item` identifiable like in SwiftUI's alert?
```suggestion
    func toast<Item>(item: Binding<Item?>, duration: Double = 2, tapToDismiss: Bool = true, offsetY: CGFloat = 0, alert: @escaping (Item?) -> AlertToast, onTap: (() -> ())? = nil, completion: (() -> ())? = nil) -> some View where Item : Identifiable {
```

**Review by nathanfallet — COMMENTED** [review]
Would love to see this merged as using an item is a common use

**Comment by elai950** [comment]
@Yvent @nathanfallet 
Can you confirm that there is no compile error or any other bugs ?

**Review by nathanfallet — APPROVED** [review]

**Comment by nathanfallet** [comment]
@elai950 I confirm, I'm using this in my app as custom code and it works.

**Pull request merged**